backend/app/routers/solver.py

```
@@ -0,0 +1,220 @@
+                full_response += chunk
+                yield f"data: {json.dumps({'chunk': chunk, 'session_id': str(session_id)})}\n\n"
+            
+            # Finalize Answer in DB
+            async with SessionLocal() as async_db:
+                result = await async_db.execute(select(Answer).where(Answer.question_id == new_question.id))
+                answer_rec = result.scalar_one_or_none()
+                if answer_rec:
+                    answer_rec.content = full_response
+                    answer_rec.status = "done"
+                    await async_db.commit()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)